### PR TITLE
Add ParadeDB and its extensions

### PR DESCRIPTION
**Pull request by philippemnoel**
Hi!

This PR adds ParadeDB and its two main extensions, which can also be used independently of ParadeDB itself. Thank you for maintaining this repository and for including us!

- [x] I've read and understood [Contributing Guidelines](CONTRIBUTING.md).
- [x] I've added the new resource sorted alphabetically in its section.
- [x] This resource is out there for a while, and actively maintained.
- [x] This resource is popular enough and has at least a few hundred stars on GitHub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,6 +235,7 @@ for `EXPLAIN`, that also provides performance tips (Commercial Software).
 ### Docker images
 * [citusdata/citus](https://hub.docker.com/r/citusdata/citus/) - Citus official images with citus extensions. Based on the official Postgres container.
 * [mdillon/postgis](https://hub.docker.com/r/mdillon/postgis/) - PostGIS 2.3 on Postgres 9. Based on the official Postgres container.
+* [paradedb/paradedb](https:/hub.docker.com/r/paradedb/paradedb/) - ParadeDB officla image with ParadeDB extensions (pg_bm25, pg_analytics).
```
Comment (jankatins) [inline README.md:238]:
```suggestion
* [paradedb/paradedb](https:/hub.docker.com/r/paradedb/paradedb/) - ParadeDB offical image with ParadeDB extensions (pg_bm25, pg_analytics).
```

File: README.md [hunk 2]
```diff
@@ -124,6 +124,8 @@ For Database Management
 * [Citus](https://github.com/citusdata/citus) - Scalable PostgreSQL cluster for real-time workloads.
 * [cstore\_fdw](https://github.com/citusdata/cstore_fdw) - Columnar store for analytics with PostgreSQL.
 * [cyanaudit](https://pgxn.org/dist/cyanaudit/) - Cyan Audit provides in-database logging of all DML activity on a column-by-column basis.
+ * [pg_analytics](https://github.com/paradedb/paradedb/tree/dev/pg_analytics) - pg_analytics is an extension that accelerates analytical query processing inside Postgres to a performance level comparable to dedicated OLAP databases.
```
Comment (jankatins) [inline README.md:127]:
```suggestion
* [pg_analytics](https://github.com/paradedb/paradedb/tree/dev/pg_analytics) - pg_analytics is an extension that accelerates analytical query processing inside Postgres to a performance level comparable to dedicated OLAP databases.
```

**Comment by philippemnoel** [comment]
Apologies for the typo, thank you for suggesting -- applied!

**Comment by philippemnoel** [comment]
Hi @jankatins Just wanted to double check here, please let me know if anything else is holding back this PR

**Comment by philippemnoel** [comment]
Hello! Just wanted to double check on this @jankatins

**Comment by jankatins** [comment]
@philippemnoel: Just FYI, I'm not a maintainer, I just reviewed the PR because I was curious and happened to notice the items.

**Comment by philippemnoel** [comment]
Ah, got it. Well, @dhamaniasad any chance you'd be able to take a look please?

**Comment by philippemnoel** [comment]
> @philippemnoel can you add a description of what ParadeDB is?

Absolutely, done.

**Pull request merged**